### DISCUSSION TITLE
LXC of Guacamole

### DISCUSSION THREAD

**Discussion by thytetgc** [tteck/Proxmox#3385] [Ideas]
It would be really cool to have Guacamole installation automated!

**Comment by SoTHISIsFitness** [comment]
if there were a tteck script for adding Guacamole to an existing LXC (rather than creating a new LXC for Guacamole) that would be amaaaaazing (I think that's what you probably meant?)

**Comment by captioustech** [comment]
I used a deb template and ran this https://github.com/itiligent/Guacamole-Install
 
works good, updates well. not quite as easy as tteck's stuff but there are node level updaters.
